src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java

```diff
@@ -0,0 +1,74 @@
+package edu.greenblitz.tobyDetermined.commands.swerve;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class MoveToPos extends SwerveCommand {
+    private ProfiledPIDController xController;
+    private ProfiledPIDController yController;
+    private ProfiledPIDController rotationController;
+    private Pose2d pos;
+
+    public static final double rotKp = 0.2;
+    public static final double rotKi = 0;
+    public static final double rotKd = 0;
+
+    public static final double translationKp = 0.2;
+    public static final double translationKi = 0;
+    public static final double translationKd = 0;
+
+    public MoveToPos(Pose2d pos) {
+        SmartDashboard.putBoolean("did push", true);
+        this.pos = pos;
+
+        xController = new ProfiledPIDController(translationKp,translationKi,translationKd,RobotMap.Swerve.Autonomus.constraints);
+        yController = new ProfiledPIDController(translationKp,translationKi,translationKd,RobotMap.Swerve.Autonomus.constraints);
+        rotationController = new ProfiledPIDController(rotKp,rotKi,rotKd,RobotMap.Swerve.Autonomus.constraints);
+        rotationController.enableContinuousInput(-Math.PI,Math.PI);
+        xController.setTolerance(0.2); //magic number and proud
```

Comment:
Broder - you shouldn't be proud, put this in const (translation tolerance + rotation tolerance)